Documents/Stephen_Workspace/Baltimore EPA.ipynb

```diff
@@ -502,6 +502,49 @@
     "display(m)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "23672919",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "FileNotFoundError",
```

Comment:
What is this error for?

Reply:
This was for when I had to interrupt generating a map because the file was too big and crashed the Jupyter notebook. I will clean that up soon.

Reply:
Ok, I'll go ahead and approve this now.